WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -993,6 +749,32 @@ public void onInputUpdated(String input) {
         dialog.show(getFragmentManager(), null);
     }
 
+    private void showLocationPicker() {
+        if (!isAdded()) {
+            return;
+        }
+        PlacePicker.IntentBuilder builder = new PlacePicker.IntentBuilder();
+        // Pre-pick the previous selected location if any
+        LatLng latLng = null;
+        if (mPostLocation != null) {
```

Comment:
We'll be updating the post object immediately after a change as you can see in #6082 for many other properties.